lib/resources/r4/related_person.yaml

```diff
@@ -229,7 +229,7 @@ fields:
   action: create
   note: |
     <ul>
-      <li>If provided, the <code>valueCodeableConcept.coding[0].code</code> must be `Encounter`.</li>
+      <li> The <code>valueCodeableConcept.coding[0].code</code> can be `Encounter or Patient`. </li>
```

Comment:
nit: let's format this so that only the values are in code blocks. i.e.
```
...  can be `Encounter` or `Patient`.
```

Reply:
+1

Reply:
updated: https://github.com/cerner/fhir.cerner.com/pull/689/commits/67460faa20177e8dab6a8154cd51a4930b8c74de#diff-261586474bd9faacdccf38c13eea61175a7bba5b7ce8ddb4dea273142eb9f6d6R232